docs/how-to/create-beamline.rst

```diff
@@ -68,12 +81,17 @@ The following example creates a fictitious beamline ``w41``, with a simulated tw
             bl_prefix=False,
         )
 
-    def d11(name: str = "D11") -> AdAravisDetector:
-        """
-        Also a valid Device factory function, but as multiple calls would instantiate
-        multiple copies of a device, discouraged.
-        """
-        return AdAravisDetector(name=name, prefix=f"{BL}-DI-DCAM-01:")
```

Comment:
Should: Now that you've removed the detector the text above about having a synchrotron and a detector doesn't make sense. If we don't want to use the `AdAravisDetector` as an example we should have some example of a device instantiated w/o `skip_device`

Reply:
I've restored the Detector. I had originally thought of showing the pros and cons of all of the methods currently available, and using the Synchrotron for all of them to make it clear what was going on, but I think you're right and we don't need to highlight the outdated ways of doing things.